### PR TITLE
Container Template: Parse object_labels

### PR DESCRIPTION
**Pull request by zakiva**
Depends on: https://github.com/ManageIQ/manageiq/pull/15406
@simon3z @moolitayer Please review

### CODE REVIEW

**Comment by zakiva** [comment]
@miq-bot add_label enhancement

File: spec/models/manageiq/providers/openshift/container_manager/refresh_parser_spec.rb [hunk 1]
```diff
@@ -420,14 +420,22 @@ def parse_entities(namespace_pod, namespace_config)
                               'required'    => true
                              }
                            ],
-                           :objects    => []
+                           :objects    => [],
+                           :labels     => {
```
Comment (cben) [inline spec/models/manageiq/providers/openshift/container_manager/refresh_parser_spec.rb]: can you also add an example of regular labels in this test?

Comment (zakiva) [reply spec/models/manageiq/providers/openshift/container_manager/refresh_parser_spec.rb]: done

**Comment by cben** [comment]
LGTM again

**Comment by zakiva** [comment]
@simon3z Please review

**Pull request merged**